src_fun/run.cpp

```diff
@@ -276,6 +276,16 @@ static bool run_intrinsic(User* user, Unit* unit, byte* storage, Block* block, S
         child_env->puppeteer_event_is_actionable = false;
         child_env->puppeteer_event = { PIPELINE_TASK_RUN, unit };
     }
+    else if (intrinsic == "test_assert"_s)
+    {
+        bool* condition_ptr;
+        get_runtime_parameter("condition"_s, &condition_ptr);
+        if (!(*condition_ptr))
+        {
+            fprintf(stderr, "Assertion failure!\n");

```

Comment:
Add an `@Incomplete` to add location information to the error. But why is this not implemented in fun? Check comment in testing.cpp